MagickCore/CMakeLists.txt

```diff
@@ -240,7 +238,7 @@ set(MAGICKCORE_SRC
   utility.h
   utility-private.h
   version.c
-  version.h
+  # version.h
```

Comment:
Is this intended? And if so just remove?

Reply:
That was intentional. That file is used, but it's generated and not present in the source dir. This silences a CMake warning.